src/handlers/positions/handler.js

```diff
@@ -248,54 +247,28 @@ const positions = async (error, messages) => {
   }
 }
 
-const createPositionHandlers = async (participantName) => {
-  try {
-    const positionHandler = {
-      command: positions,
-      config: Utility.getKafkaConfig(Utility.ENUMS.CONSUMER, TransferEventType.POSITION.toUpperCase(), TransferEventAction.PREPARE.toUpperCase())
-    }
-    const topicNameList = [
-      Utility.transformAccountToTopicName(participantName, TransferEventType.POSITION, TransferEventAction.ABORT),
-      Utility.transformAccountToTopicName(participantName, TransferEventType.POSITION, TransferEventType.FULFIL),
-      Utility.transformAccountToTopicName(participantName, TransferEventType.POSITION, TransferEventAction.PREPARE)
-    ]
-    await Kafka.Consumer.createHandler(topicNameList, positionHandler.config, positionHandler.command)
-  } catch (error) {
-    Logger.error(error)
-    throw error
-  }
-}
-
 /**
- * @function RegisterPositionsHandlers
+ * @function registerPositionHandler
  *
  * @async
- * @description Registers the position handlers for all participants. Retrieves the list of all participants from the database and loops through each
- * createPositionHandler called to create the handler for each participant
- * @param {string[]} participantNames - Array of Participants to register
+ * @description Registers the handler for position topic. Gets Kafka config from default.json
+ *
  * @returns {boolean} - Returns a boolean: true if successful, or throws and error if failed
  */
-const registerPositionHandlers = async (participantNames = []) => {
+const registerPositionHandler = async () => {
   try {
-    let participantNamesList
-    if (Array.isArray(participantNames) && participantNames.length > 0) {
-      participantNamesList = participantNames
-    } else {
-      participantNamesList = await DAO.retrieveAllParticipants()
-    }
-    if (participantNamesList.length !== 0) {
-      for (let name of participantNamesList) {
-        await createPositionHandlers(name)
-      }
-    } else {
-      Logger.info('No participants for position handler creation')
-      return false
+    const positionHandler = {
+      command: positions,
+      topicName: Utility.transformGeneralTopicName(TransferEventType.POSITION, TransferEventAction.PREPARE),
+      config: Utility.getKafkaConfig(Utility.ENUMS.CONSUMER, TransferEventType.TRANSFER.toUpperCase(), TransferEventAction.POSITION.toUpperCase())
     }
-  } catch (error) {
-    Logger.error(error)
-    throw error
+    positionHandler.config.rdkafkaConf['client.id'] = positionHandler.topicName
```

Comment:
this should not be overwritten. take the client.id from config and concat a uuid to it